components/login/passwordReset/src/index.scss

```diff
@@ -0,0 +1,5 @@
+@import '~@s-ui/theme/lib/index';
+
+.sui-LoginPasswordReset {
```

Comment:
```suggestion
.mt-LoginPasswordReset {
```

🍆  should be like this, right?

Reply:
This component is intended to be global. reusable for all verticals. Maybe we can think if the prefix "sui-" is the best... but coupling it to "mt" is not the idea.

Reply:
you're absolutely right! I missed we're on this repo! 😅